src/common/test-runtime/project.json

```diff
@@ -0,0 +1,43 @@
+{
+  "dependencies": {
```

Comment:
right, Packaging tests use to reference the one inside Tools dir, but that one  is outdated which is why I believe she is mimicking what we have in corefx where we have now our own copy of it that we know works with it. Also, this is the reason why she changed global.json